webpack.config.js

```diff
@@ -0,0 +1,84 @@
+const { exec } = require('child_process')
+const path = require('path')
+const { EsbuildPlugin } = require('esbuild-loader')
+
+const webpack = require('webpack')
+
+// Run `yarn start.plugins` to enable debug mode.
+// This mode will serve the plugin bundle via a local dev-server.
+const debug = process.env.WEBPACK_SERVE
+
+// Try exposing our socket to adb (errors are fine):
+if (process.env.WEBPACK_SERVE) {
+  console.log('adb reverse tcp:8101 tcp:8101')
+  exec('adb reverse tcp:8101 tcp:8101', () => {})
```

Comment:
This port, 8101, the the one the GUI currently uses. That is fine!

However, edge-core-js uses 8080, edge-currency-accountbased used 8082, and edge-exchange plugins uses 8083. So it seems like 8081 would "fit" better than 8101.